rest/rest_block.py

```diff
@@ -174,6 +176,14 @@ def _locked_poll(self, paging=False):
             try:
                 signals, paging = self._process_response(resp)
                 signals = self._discard_duplicate_posts(signals)
+
+                # add the include_query attribute if it is configured
+                if self.include_query is not None and signals is not None:
```

Comment:
If `include_query` is an empty string, we probably don't want to include it either. That sound right?

Right now, it will only not include if it's None